### PR TITLE
Fix/133898 datepicker pro code

### PR DESCRIPTION
**Pull request by paulmskim**
https://central.tri.be/issues/133898

https://www.loom.com/share/351b882414ec45e886bea0b08dacf42b

### CODE REVIEW

File: src/resources/js/views/datepicker.js [hunk 1]
```diff
@@ -65,6 +57,39 @@ tribe.events.views.datepicker = {};
...
+	 *
+	 * @since TBD
+	 *
+	 * @type {PlainObject}
+	 */
+	obj.options = {
+		container: null,
+		daysOfWeekDisabled: [],
+		/**
+		 * @todo: use format from BE. Paul.
```
Comment (bordoni) [inline src/resources/js/views/datepicker.js]: Lets start using the `@` from github on these.

File: src/resources/js/views/datepicker.js [hunk 2]
```diff
@@ -363,61 +314,49 @@ tribe.events.views.datepicker = {};
...
+		// set up mutation observer
+		obj.observer = new MutationObserver( obj.handleMutation( { container: $container } ) );
+
+		// set options for datepicker
+		obj.options.container = $container.find( obj.selectors.datepickerContainer );
+		obj.options.minViewMode = isMonthView ? 'year' : 'month';
+		var tribeL10nDatatables = window.tribe_l10n_datatables || {};
+		var datepickerI18n = tribeL10nDatatables.datepicker || {};
+		var nextText = datepickerI18n.nextText || 'Next';
+		var prevText = datepickerI18n.prevText || 'Prev';
```
Comment (bordoni) [inline src/resources/js/views/datepicker.js]: No need for this extra string here. It should always come from the localize var.

Comment (paulmskim) [reply src/resources/js/views/datepicker.js]: You're right, we should always have this available, but the 2 lines above I've set empty objects if in the off case `tribe_l10n_datatables` does not exist. So `nextText` and `prevText` from `datepickerI18n` object will be undefined. `Next` and `Prev` are fallbacks.

File: src/resources/js/views/datepicker.js (hunk 1 above)
Comment (paulmskim) [reply src/resources/js/views/datepicker.js]: copied these from other places, but will add.

**Pull request merged**